SPECS/openssl/openssl.spec

```diff
@@ -137,7 +137,6 @@ export HASHBANGPERL=/usr/bin/perl
     no-sm4 \
     no-ssl \
     no-ssl3 \
-    no-tests \
```

Comment:
Sync with Andrew and make sure to run a buddy-build for this.  We want to be sure that the test sections do not pull source from the network at build time.  It's probably ok...but we need to be sure.  Alternativel you can wrap this in a macro so that it is only configured for test builds.

Reply:
I think %check is already wrapped with macro to enable package test when required and by default it is disabled. I think we don't need other macros for disabling the package test run.

Reply:
I think Jon is referring to the possibility of the removal of `no-test` to try and pull something over the network during `%build`, not `%check`.

Reply:
Executed buddy pipeline and no issue was seen.